src/java.base/share/classes/com/alibaba/util/QuickStart.java

```diff
@@ -125,6 +126,39 @@ private static void initialize(int roleCode, String cachePath, boolean verbose,
         if (role == QuickStartRole.TRACER || role == QuickStartRole.PROFILER) {
             Runtime.getRuntime().addShutdownHook(new Thread(QuickStart::notifyDump));
         }
+        if (role == QuickStartRole.PROFILER) {
+            String startupProbe = System.getenv("DRAGONWELL_QUICKSTART_STARTUP_PROBE");
+            if (startupProbe != null && !startupProbe.equals("")) {
+                Thread t = new Thread( new Runnable() {
+                    @Override
+                    public void run() {
+                        runStartupDump();
+                    }
+                },"Startup-Probe");
+                t.setDaemon(true);
+                t.start();
+            }
+        }
+    }
+
+    private static void runStartupDump() {
+        String jdkHome = Utils.getJDKHome();
+        Utils.runProcess(verbose, "[StartupProbe] ", (pb)-> {
```

Comment:
Explicitly catch and handle the exception. otherwise, the error will be handled by `Thread.UncaughtHandler`

Reply:
Fixed